resources/ory/files/secret-migration.sh

```diff
@@ -74,36 +80,38 @@ if [[ -z "${COOKIE}" ]]; then
   COOKIE=$(get_from_file "${SECRET_COOKIE_KEY}" || generateRandomString 32)
 fi
 
+{{- if contains .Values.postgresql.replication.enabled "true" }}
 DATA=$(cat << EOF
   ${DSN_KEY}: $(echo -n "${DSN}" | base64 -w 0)
   ${SECRET_SYSTEM_KEY}: $(echo -n "${SYSTEM}" | base64 -w 0)
   ${SECRET_COOKIE_KEY}: $(echo -n "${COOKIE}" | base64 -w 0)
   {{- if .Values.global.ory.hydra.persistence.enabled }}
   ${PASSWORD_KEY}: $(echo -n "${PASSWORD}" | base64 -w 0)
+  ${PASSWORD_R_KEY}: $(echo -n "${PASSWORDR}" | base64 -w 0)
```

Comment:
i think `if else` can be wrapped only around this line or am i missing something?

Reply:
yup its better thank you for your comment, I'll update.